src/fare-zones-selector/FareZoneResults.tsx

```diff
@@ -3,69 +3,69 @@ import {Pin} from '@atb/assets/svg/mono-icons/map/';
-import {getReferenceDataName, TariffZone} from '@atb/modules/configuration';
+import {getReferenceDataName, FareZone} from '@atb/modules/configuration';
 import {StyleSheet} from '@atb/theme';
 import {
   getTextForLanguage,
-  TariffZoneSearchTexts,
+  FareZoneSearchTexts,
   useTranslation,
 } from '@atb/translations';
 import {insets} from '@atb/utils/insets';
 import React from 'react';
 import {ScrollView, View} from 'react-native';
-import {useTariffZoneFromLocation} from '@atb/tariff-zones-selector/use-tariff-zone-from-location';
+import {useFareZoneFromLocation} from '@atb/fare-zones-selector/use-fare-zone-from-location';
 import {PressableOpacity} from '@atb/components/pressable-opacity';
 
 type Props = {
-  tariffZones: TariffZone[];
-  onSelect: (tariffZone: TariffZone) => void;
+  fareZones: FareZone[];
+  onSelect: (fareZone: FareZone) => void;
 };
 
-export const TariffZoneResults: React.FC<Props> = ({tariffZones, onSelect}) => {
+export const FareZoneResults: React.FC<Props> = ({fareZones, onSelect}) => {
   const styles = useThemeStyles();
   const {t, language} = useTranslation();
-  const tariffZoneFromLocation = useTariffZoneFromLocation(tariffZones);
+  const fareZoneFromLocation = useFareZoneFromLocation(fareZones);
   return (
     <>
       <View accessibilityRole="header" style={styles.subHeader}>
         <ThemeText typography="body__secondary" color="secondary">
-          {t(TariffZoneSearchTexts.zones.heading)}
+          {t(FareZoneSearchTexts.zones.heading)}
         </ThemeText>
       </View>
       <ScrollView>
-        {tariffZones.map((tariffZone) => (
-          <View style={styles.rowContainer} key={tariffZone.id}>
-            <View style={styles.tariffZoneButtonContainer}>
+        {fareZones.map((fareZone) => (
+          <View style={styles.rowContainer} key={fareZone.id}>
+            <View style={styles.fareZoneButtonContainer}>
               <PressableOpacity
                 accessible={true}
                 accessibilityLabel={
                   t(
-                    TariffZoneSearchTexts.zones.item.a11yLabel(
-                      getReferenceDataName(tariffZone, language),
+                    FareZoneSearchTexts.zones.item.a11yLabel(
+                      getReferenceDataName(fareZone, language),
                     ),
                   ) + screenReaderPause
                 }
-                accessibilityHint={t(TariffZoneSearchTexts.zones.item.a11yHint)}
+                accessibilityHint={t(FareZoneSearchTexts.zones.item.a11yHint)}
                 accessibilityRole="button"
                 hitSlop={insets.symmetric(8, 1)}
-                onPress={() => onSelect(tariffZone)}
-                style={styles.tariffZoneButton}
-                testID={'tariffZone' + tariffZone.name.value + 'Button'}
+                onPress={() => onSelect(fareZone)}
+                style={styles.fareZoneButton}
+                testID={'fareZone' + fareZone.name.value + 'Button'}
```

Comment:
Probably need to update tests in `e2e/` when you change testIDs.

Reply:
I've updated stuff on the e2e purchase overview part 👍🏼